### PR TITLE
[ALLUXIO-2605]Deprecate Throwables.propagate with RuntimeException in Configuration.java

### PR DESCRIPTION
**Pull request by ddzhouxm**
https://alluxio.atlassian.net/browse/ALLUXIO-2605

### CODE REVIEW

**Comment by ddzhouxm** [comment]
https://alluxio.atlassian.net/browse/ALLUXIO-2605

**Comment by ddzhouxm** [comment]
alluxio-bot, check this please

File: core/common/src/main/java/alluxio/Configuration.java [hunk 1]
```diff
@@ -396,7 +396,9 @@ public static long getMs(PropertyKey key) {
       return clazz;
     } catch (Exception e) {
       LOG.error("requested class could not be loaded: {}", rawValue, e);
-      throw Throwables.propagate(e);
+//    throw Throwables.propagate(e);
```
Comment (gpang) [inline core/common/src/main/java/alluxio/Configuration.java]: You can remove this line.

**Comment by yupeng9** [comment]
@ddzhouxm any updates?

**Comment by yupeng9** [comment]
@ddzhouxm feel free to reopen this PR after you address the comments.

**Pull request closed without merging**